docs/api/data-views/swap_references.asciidoc

```diff
@@ -0,0 +1,75 @@
+[[data-views-api-swap-references]]
+=== Swap references data view API
+++++
+<titleabbrev>Swap references</titleabbrev>
+++++
+
+Swap saved object references
+
+CAUTION! MISUSE CAN BREAK LARGE NUMBERS OF SAVED OBJECTS! PRACTICE WITH A BACKUP!
+
+[NOTE]
+====
+For the most up-to-date API details, refer to the
+{kib-repo}/tree/{branch}/src/plugins/data_views/docs/openapi[open API specification].
+====
+
+[[data-views-api-swap-references-request]]
+==== Request
+
+`POST <kibana host>:<port>/api/data_views/swap_references`
+
+`POST <kibana host>:<port>/s/<space_id>/api/data_views/swap_references`
+
+
+[[data-views-api-swap-references-request-body]]
+==== Request body
+
+fromId - Saved object reference to change
+
+toId - New saved object reference value to replace the old
+
+delete - Deletes referenced saved object if all references are removed
+
+fromType - Specify the type of the saved object reference to alter. Default is `index-pattern` for data view
+
+forId - Limit the affected saved objects to one or more by id
+
+forType - Limit the affected saved objects by type
```

Comment:
```suggestion
`formId`::
(Required, string) Saved object reference to change.

`toId`::
(Required, string) New saved object reference value to replace the old.

`delete`::
 (Optional, boolean) Deletes referenced saved object if all references are removed.

`fromType`::
 (Required, object) Specify the type of the saved object reference to alter. Default is `index-pattern` for data view.

`forId`::
(Optional, string) Limit the affected saved objects to one or more by IDs.

`forType`::
 (Optional, string) Limit the affected saved objects by type.
```

Reply:
Pretty please check all my prefixes -> (Optional, string) and so on because I've probably made some mistakes with them.